### ISSUE TITLE
Implementing Configuration

### ISSUE THREAD

**Issue by Nitzantomer1998** [Nitzantomer1998/UniversalHebrewSubtitles#5]
As more sources are added to the addon, it is essential to provide users with the ability to choose which sources to use for finding subtitles. This feature is crucial for greater customization and accuracy, ensuring that users can select the most reliable and relevant subtitle sources for their needs.

**Comment by Nitzantomer1998** [comment]
The configuration has been successfully implemented.

**Comment by TwilightMercy** [comment]
Looks very nice!

![image](https://github.com/Nitzantomer1998/UniversalHebrewSubtitles/assets/29895680/d7403ac3-0323-460d-ab48-3dd6b738e910)

**Comment by Nitzantomer1998** [comment]
Thank you, its not final by the way.
Just waned to finish a mockup and continue to implement OpenSubtitles which just been upload.

**Comment by TwilightMercy** [comment]
Will the link stay "wizdom" or will be changed to "universal-hebrew-subtitles" / something like that?

**Comment by Nitzantomer1998** [comment]
Hi, @TwilightMercy,

I've just made some minor adjustments to the configure page. Please take a look and let me know your thoughts. If you have any suggestions for improvement or if you see a better layout, feel free to share them.

**Comment by TwilightMercy** [comment]
Looks better and cleaner! (I've noticed the change in "Save" button look),

I suggest:
1. Maybe add the [addon logo](https://github.com/Nitzantomer1998/UniversalHebrewSubtitles/blob/main/public/icon.svg) somewhere in the top of the page

2. I've added the (current) addon name to header, added direct URLs to each subtitles website, and changed "Save" button label to "Install". Feel free to try if you like it:
https://pastebin.com/raw/K32hexu6
